src/liquid/components/ld-bg-cells/readme.md

```diff
@@ -302,13 +315,13 @@ A background pattern with the Merck cells as additional visual element.
 <!-- React component -->
 
 <LdBgCells
-  style={ {
-    '--ld-bg-cells-layer-translation-x': '26%',
-    '--ld-bg-cells-layer-translation-y': '-5%',
-    '--ld-bg-cells-layer-size': '195%',
-    aspectRatio: 1,
-  } }
- />
+style={ {
+'--ld-bg-cells-layer-translation-x': '26%',
+'--ld-bg-cells-layer-translation-y': '-5%',
+'--ld-bg-cells-layer-size': '195%',
+aspectRatio: 1,
+} }
+/>
```

Comment:
Please fix the indentation, it might be that the linter/formatter removes it, but it should be preserved for the code examples. If necessary, save without formatting after fixing the indent. ([cmd] + [k] and then [s] should do the trick in vscode.)